Misc/NEWS.d/next/Library/2021-06-20-19-01-11.bpo-44404.McfrYB.rst

```diff
@@ -0,0 +1 @@
+tkinter ``after`` now supports callable classes
```

Comment:
```suggestion
:mod:`tkinter`'s ``after()`` method now supports callables without the ``__name__`` attribute.
```

Reply:
Based on this wording, is it worth ignoring a potential `AttributeError` raised by the `type(func).__name__` line? This will almost always succeed, won't it (I'm not sure of the situation if metaclasses are used)?

Reply:
Having tried it I don't believe `type(func).__name__` can fail and therefore a check would be unnecessary. Committing suggestion.